packages/bygger/server/src/Backend.ts

```diff
@@ -11,35 +11,38 @@ import {
   performChangesOnSeparateBranch,
   pushFilesAndUpdateSubmoduleCallback,
 } from "./repoUtils.js";
-import formioService from "./services/formioService";
+import { FormioService } from "./services/formioService";
 
 export class Backend {
   private readonly skjemaUtfylling: GitHubRepo;
   private readonly config: ConfigType;
+  private readonly formioService: FormioService;
 
-  constructor(config: ConfigType) {
+  constructor(config: ConfigType, formioService: FormioService) {
     this.config = config;
+    this.formioService = formioService;
     this.skjemaUtfylling = new GitHubRepo(config.publishRepo.owner, config.publishRepo.name, config.publishRepo.token);
   }
 
-  async publishForm(formContent: NavFormType, translationsContent: I18nTranslations, formPath: string) {
+  async publishForm(formContent: NavFormType, translationsContent: I18nTranslations | undefined, formPath: string) {
```

Comment:
Hent formPath fra formContent, og gjør translationsContent optional i stedet for å tillate undefined